### PR TITLE
Mc eateries

### PR DESCRIPTION
**Pull request by ghost**
git fetch --all
git checkout mc-eateries

### CODE REVIEW

File: scripts/bizarries/BizSelect.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { useBiz, getBiz } from "./BizProvider.js";
 
-const contentTarget = document.querySelector(".container__select-list")
```
Comment (heymonicakay) [inline scripts/bizarries/BizSelect.js:3]: Perfect!

File: scripts/bizarries/BizList.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 import { getBiz, useBiz } from "./BizProvider.js";
 import { BizHTML } from "./BizHTMLGenerator.js";
 
-const contentTarget = document.querySelector(".bizarries")
+const contentTarget = document.querySelector(".container--biz")
```
Comment (heymonicakay) [inline scripts/bizarries/BizList.js:5]: Lovely!

File: scripts/eateries/EateryList.js [hunk 3]
```diff
@@ -1,5 +1,36 @@
...
+
+eventHub.addEventListener("eaterySelected", event => {
+
+    const eateryThatWasSelected = event.detail.eateryId
+    const allEateries = useEatery()
+    const foundEatery = allEateries.find(eateryObj => {
+        return eateryObj.id === (parseInt(eateryThatWasSelected))
+    })
+    console.log(foundEatery)
+    render(foundEatery)
```
Comment (heymonicakay) [inline scripts/eateries/EateryList.js:16]: Confirmed eatery dropdown renders, and the eatery selected populates within the itinerary preview after selecting an eatery. Great work!

**Review by heymonicakay — APPROVED** [review]
Approved

**Review by ardisrtilson — APPROVED** [review]

**Pull request merged**